src/lib/commandCenter/panels/ai.svelte

```diff
@@ -15,9 +15,9 @@
     const { input, handleSubmit, completion, isLoading, complete, error } = useCompletion({
         api: endpoint + '/console/assistant',
         headers: {
-            'x-appwrite-mode': 'admin',
```

Comment:
The project ID should be added to the request header.

In addition, the [change](https://github.com/appwrite/appwrite/blob/557513c3eefd5e5b98be49ddd86835a43d455041/app/init.php#L1045) that made Appwrite default to the console project needs to be removed  because of https://github.com/appwrite/appwrite/commit/f7f14695f0943b101280d3f880348b74307d1060

Reply:
Set project id to console here.

> Why shouldn't it default to console?

https://github.com/appwrite/appwrite/commit/f7f14695f0943b101280d3f880348b74307d1060